i18n/GLOSSARY_ru.md

```diff
@@ -0,0 +1,49 @@
+## GitOps-Глоссарий {{version}}
+
+Этот глоссарий дополняет [Принципы GitOps](./PRINCIPLES_ru.md) и другие сопроводительные документы в этом репозитории.
+
+- ## Непрерывность
+
+    Под «непрерывным» в контексте [согласования](#согласование) понимается, что оно происходит регулярно, но не обязательно немедленно.
+
+- ## Декларативное описание
+
+    Конфигурация, описывающая желаемое рабочее состояние системы без указания процедур для достижения этого состояния. Это разделяет конфигурацию (желаемое состояние) и реализацию (команды, вызовы API, скрипты и т.д.), используемую для достижения этого состояния.
+
+- ## Желаемое состояние
+
+    Совокупность всех конфигурационных данных, достаточных для воссоздания системы таким образом, чтобы её экземпляры были неразличимы с точки зрения поведения.
+    Эти конфигурационные данные, как правило, не включают постоянные данные приложений, например, содержимое базы данных, но часто включают учетные данные для доступа к этим данным или конфигурации для инструментов восстановления данных, работающих на этой системе.
+
+- ## Отклонение (Drift)
+
+    Когда фактическое состояние системы отклонилось или начинает отклоняться от желаемого состояния.
+
+- ## Автоматическая вытягиваемoсть (Pull)
+    
+    **Принцип 3** ([Автоматически вытягиваемый](./PRINCIPLES_ru.md)) указывает, что желаемое состояние должно быть «вытянуто» (*pull*), а не «иньецировано» (*push*), главным образом потому, что программные агенты должны иметь возможность в любой момент времени получать доступ к желаемому состоянию из хранилища состояния, а не только когда происходит намеренное изменение в хранилище состояния, вызывающее событие *push*. Это является необходимым условием для того, чтобы согласование происходило непрерывно, как указано в **принципе 4** (Непрерывное согласование). Обратите внимание, что – в отличие от традиционного CI/CD, где автоматизация обычно запускается заранее заданными триггерами – в GitOps согласование запускается в любое время, когда происходит расхождение. Расхождение может быть связано с тем, что фактическое состояние непреднамеренно отклоняется от деклараций желаемого состояния, а не только из-за того, что была намеренно изменена версия декларации нового желаемого состояния.
```

Comment:
Regarding usage of "инъецировано" : I tried to do a little bit of search for the official term for "push" in Russian and I'm wondering if "отправлено"/"передано" would be a better alternative here. I think that "инъецировано" is basically "triggered" but it might be misunderstood (potentially) since a pull-event also triggers changes. I would consider to change this to be closer to "push" definition at least.

Reply:
I was thinking about this one too. I chodes injected/иньецировано just to make the contrast to pull. 
"отправлено"/"передано" don't have it in them IMHO.  The straightforward translation to  of "pull" in this context is probably "толкать, втолкать", but sounds wired to me in Russian. 

What to you think if we elaborate your suggestion and  say:  " а не «передано из вне» (*push*)"

Reply:
Check lastes change.